### PR TITLE
Fix Broken Link

### LINKED ISSUE #20495: Broken Link OpenCV
##### System information (version)

- OpenCV => 4.2
- Operating System / Platform => Windows 64 Bit, Linux

##### Detailed description
Link no longer exist : 
1. OpenCV/3rdparty/readme.txt => http://www.remotesensing.org/libtiff/

Can be replaced with the new address found at: http://www.libtiff.org/

##### Steps to reproduce
link1

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [ ] I checked the problem with documentation, FAQ, open issues,
       forum.opencv.org, Stack Overflow, etc and have not found solution
   
 - [ ] I updated to latest OpenCV version and the issue is still there
   
 - [ ] There is reproducer code and related data files: videos, images, onnx, etc

### PR DESCRIPTION
**Pull request by abhi-bhatra**
resolves #20495

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [ ] I agree to contribute to the project under Apache 2 License.
- [ ] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [ ] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

Details:
Fixed the links:
-> Broken/Missing Links
-> Some links expired

### CODE REVIEW

File: 3rdparty/readme.txt [hunk 1]
```diff
@@ -51,7 +51,9 @@ jasper                JasPer is a collection of software
                       Copyright (c) 1999-2000 The University of British Columbia
                       Copyright (c) 2001-2003 Michael David Adams
 
-                      The JasPer license can be found in libjasper.
+                      See jasper official github repository
```
Comment (alalek) [inline 3rdparty/readme.txt:54]: Use proper names:
- JasPer
- GitHub

**Review by alalek — COMMENTED** [review]
Thank you for contribution!

This patch should go into [3.4 branch](https://github.com/opencv/opencv/wiki/Branches) first.
We will merge changes from 3.4 into master regularly (weekly/bi-weekly).

Please:
- change "base" branch of this PR: master => 3.4 (use "Edit" button near PR title)
- rebase your commits from master onto 3.4 branch. For example:
    `git rebase -i --onto upstream/3.4 upstream/master`
    (check list of your commits, save and quit (Esc + "wq" + Enter)
    where `upstream` is configured by following [this GitHub guide](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and fetched (`git fetch upstream`).
- push rebased commits into source branch of your fork (with `--force` option)

Note: no needs to re-open PR, apply changes "inplace".

File: 3rdparty/readme.txt (hunk 1 above)
Comment (abhi-bhatra) [reply 3rdparty/readme.txt:54]: Hey @alalek thanks for the response
I will do the necessary changes and change the branch to 3.4 now !

**Comment by alalek** [comment]
I will rebase commits for this PR.

**Review by alalek — APPROVED** [review]
Thank you!

**Pull request merged**